api/src/services/HierarchyCollector.ts

```diff
@@ -37,43 +37,53 @@ class HierarchyCollector {
         // we can skip fetching more RDF in order to save some time!
         const RDFPromise = fetchRdf ? this.fedora.getRdf(pid) : null;
         const [DC, RELS, RDF] = await Promise.all([DCPromise, RELSPromise, RDFPromise]);
-        const dataStreams = fetchRdf ? this.extractor.extractFedoraDatastreams(RDF) : [];
         const relations = this.extractor.extractRelations(RELS);
+        const dataStreams = fetchRdf ? this.extractor.extractFedoraDatastreams(RDF) : [];
```

Comment:
Since you changed `dataStreams` to `datastreams` in the code below, maybe it makes sense to also change it in this function for consistency (I agree that all lowercase seems more appropriate, given the way we use this word).

Reply:
made change